mat_dp_pipeline/sdf/standard_data_format.py

```diff
@@ -8,25 +8,20 @@
 import numpy as np
 import pandas as pd
 
-from mat_dp_pipeline.common import FileOrPath
+from mat_dp_pipeline.common import FileOrPath, validate
 
 Year = int
 
 
-def validate(condition: bool, error_message: str | None = None):
-    if not condition:
-        raise ValueError(error_message)
-
-
-def validate_tech_units(tech_meta: pd.DataFrame):
-    if tech_meta.empty:
+def validate_tech_units(tech_metadata: pd.DataFrame) -> None:
```

Comment:
What about this `validate`? Keep or replace its usage with copy-pasta?

Reply:
I think these are fine as is - more obvious how it's used